### DISCUSSION TITLE
Pattern contribution for Empty State Pattern contribution

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1105] [Office hours] [answered]
I'm working on the specs document to contribute to the patter for Empty state pages in the Console. 

You can find the specs document [here. []()](https://docs.google.com/document/d/1IATJtMSq0tfp6oUKIE2NLKtya6xfltbLeqOQ66xXxXA/edit?usp=sharing)
Feel free to add any comments. 

Also, I wanted to ask you if you have any directions or guides for the template file?

**Comment by corinne-nagel** [comment, marked as answer]
Hi @lgalloherran thanks again for taking this pattern on 💪! I left comments in your doc and here are the templates for mockups: https://share.goabstract.com/0ee02298-df9b-4224-bebc-720d2cc6bb40

For next steps, we suggest you attend the committee meetings on Wednesdays. Let me know if you're able to make the next one (10am-12 MT on 1/27) and I can add you to the agenda. During these meetings you can present your doc, get feedback and people can review it from all crit pods, including developers.

Let us know if you have any other questions!

**Reply by serifluous** [reply]
@lgalloherran The templates we have there were just a first draft, so if you have ideas for how to make a template more usable for more PDs, definitely go ahead and do that.

**Reply by lgalloherran** [reply]
Hi @corinne-nagel  Thanks for the insights. I can't attend that meeting but I'm available for next one, so let me know if it's possible. 
Meanwhile I'll prepare the doc.

**Reply by corinne-nagel** [reply]
Sounds great! I'll add you to the agenda next week.

**Reply by corinne-nagel** [reply]
Hi @lgalloherran , checking in that you're ready to share with the committees on Wednesday? We try to send the doc out today so they can pre-read and add comments. Are you comfortable with me sending that?

**Comment by lgalloherran** [comment]
Hey Corinne,
yes, I'm working on the Sketch file because it got completely erased so I
have to redo it. But as content, I think is ready for revision.


On Mon, Feb 1, 2021 at 12:01 PM Corinne Nagel <notifications@github.com>
wrote:

> Hi @lgalloherran
> <https://urldefense.com/v3/__https://github.com/lgalloherran__;!!NCc8flgU!NXvNGhDFas9KabBwDcRO65s80wPRdQgsMVK7jjRZnFEUEP-vvjZwiPqxE9psPZyIvg$>
> , checking in that you're ready to share with the committees on Wednesday?
> We try to send the doc out today so they can pre-read and add comments. Are
> you comfortable with me sending that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.com/v3/__https://github.com/twilio-labs/paste/discussions/1105*discussioncomment-328017__;Iw!!NCc8flgU!NXvNGhDFas9KabBwDcRO65s80wPRdQgsMVK7jjRZnFEUEP-vvjZwiPqxE9o-VDPd8g$>,
> or unsubscribe
> <https://urldefense.com/v3/__https://github.com/notifications/unsubscribe-auth/ARIQPFRARMIITN6O5SIL5ULS43M5VANCNFSM4WOQAYIA__;!!NCc8flgU!NXvNGhDFas9KabBwDcRO65s80wPRdQgsMVK7jjRZnFEUEP-vvjZwiPqxE9oujwwfFw$>
> .
>


-- 
Luz Alba Gallo Herrán
Product Designer
[image: Twilio] <https://www.twilio.com/?utm_source=email_signature>
<https://www.twilio.com/?utm_source=employee_signature&utm_medium=email&utm_content=its-time-to-build>
Embark
on the next stage of your digital transformation. *Learn more.*
<https://www.twilio.com/?utm_source=employee_signature&utm_medium=email&utm_content=its-time-to-build>
